src/translations/en.json

```
@@ -2383,6 +2383,7 @@
             "name": "Name",
             "icon": "Icon",
             "level": "Level",
+            "level_helper": "Used to determine the default floor icon.",
```

I think we should be more explicit
```suggestion
            "level_helper": "Used to determine the default floor icon. Does not influence the floor order.",
```